region/norfair/west.json

```diff
@@ -1507,22 +1776,29 @@
           "to": [
             {
               "id": 2,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                {"heatFrames": 200}
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 200}
+                  ]
+                }
               ]
             },
             {
               "id": 3,
-              "requires": [
-                {"or": [
-                  "heatProof",
-                  "canHeatRun"
-                ]},
-                {"heatFrames": 150}
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "canNavigateHeatRooms",
+                    {"heatFrames": 150}
+                  ],
+                  "note": "Involves crumble block jumping. Should we have alternate strats?"
```

Comment:
The room in general is a good candidate for canWallJump.  Crumble jumping should be its own tech.  Reaching door1 would then resemble a tricky walljump.  
The intended strat to 3 looks to be either a horizontal spark from the door, or highjump+ice from door2.

Reply:
re: Crumble Jump as a tech. It started out being one, there was a discussion, it was removed, and it's been bugging me since. In fact, there was also `canCrumbleSpinJump`. I don't think we want to do that here though, because it would have to impact other regions as well.

So I created issue #284.

As for walljump, well. I see it as its own project. I wish it weren't anywhere, I just left it in where others have put it because it seemed like a shame to waste the work.

Reply:
Reaching the item I'm ok with, but I'd like to see reaching door1 be marked nontrivial

Reply:
I can do that for now. Issue #284 will end up overhauling this room completely, probably with a bunch of alternate strats.